components/Valuation/DataComparison/DataComparisonCard.tsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import { IAPIData, IPredictions } from "../../../lib/types";
+
+interface Props {
+  apiData: IAPIData | undefined;
+  predictions: IPredictions | undefined;
+}
+
+function getMetaverseTokenFromOpenseaLink(opensea_link: string | undefined) {
+  let metaverse = "";
+  let tokenID = "";
+  if (opensea_link) {
+    const split = opensea_link.split("/");
+    metaverse = split[4];
+    tokenID = split[5];
+  }
+  return { metaverse, tokenID };
+}
+
+/**
+ * 
+ * @param param0 
+ * @returns 
+ */
+const DataComparisonCard = ({ apiData, predictions }: Props) => {
+  const [offers, setOffers] = useState<number>();
+  const [openseaLink, setOpenSeaLink] = useState<string | undefined>(apiData?.external_link);
+  
+  const usdPredictionPrice = predictions?.usdPrediction;
+  const handleData = async () => {
+    let { metaverse, tokenID } = getMetaverseTokenFromOpenseaLink(
+      apiData?.opensea_link
+    );
+    const res = await fetch("/api/getLandOffers", {
+      method: "POST",
```

Comment:
Just to be consistent. The link that we use to retrieve things from openSea is actually a `GET` request, so even though it might not change things functionality wise, I would think its good to use this also as a `GET` request to keep it consistent. `POST` requests are usually for saving/adding data to a server, in here we are just fetching something

Reply:
Making it as a get also saves you lines of code right below, you don't need to send any body or headers. just `fetch(/api/getLandOffers/${metaverse}/${landID}` and thats it